_releases/release_notes_1.0.0.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+title: Release Notes 1.0.0
+nav_order: 2
+---
+
+Babelfish Version: 1.0.0
```

Comment:
Once release files are ready, I'll provide the link to the 1.0.0 release.

Reply:
Sounds good...